src/components/RegistrationTable/TableIconButtons.tsx

```diff
@@ -1,55 +1,83 @@
-import React from "react"
-import Image from 'next/image'
-import { Button } from "@/components/ui/button"
-import { TooltipProvider, Tooltip, TooltipTrigger, TooltipContent } from "@/components/ui/tooltip"
+import React, { Dispatch, SetStateAction } from "react";
+import Image from "next/image";
+import { Button } from "@/components/ui/button";
+import {
```

Comment:
I honestly dont know why my prettier config does this but the only relevant changes is having the button link to the useState of the QrReader